near/src/lib/vm/vm.js

```diff
@@ -1853,26 +1853,33 @@ export default class VM {
         },
       },
       Urbit: {
```

Comment:
Why is the `Urbit` object defined inside the return statement with `clipboard` and not outside of it in the `VM` class like `Social` and `Near`?

Reply:
No reason, I just put it in-line at first when I was messing with it. Probably makes more sense to define it with the other larger objects yeah.